tools/mo/openvino/tools/mo/subprocess_main.py

```diff
@@ -38,8 +37,15 @@ def setup_env():
     mo_root_path = os.path.join(os.path.dirname(__file__), os.pardir, os.pardir, os.pardir)
 
     # Check that MO root directory already set to the PYTHONPATH
-    status = subprocess.run([sys.executable, "-c", "try: import openvino.tools.mo \nexcept: exit(1)"], env=os.environ)
-    if status.returncode != 0:
+    def is_mo_imported():
+        try:
+            status = subprocess.run([sys.executable, os.path.join(mo_root_path, 'openvino/tools/mo/utils/check_mo_import.py')],
```

Comment:
I would you path based on current script path to avoid all this `openvino/tools/mo` subfolders.

Reply:
Communicated personally that this is nit. Will address together with other comments